### PR TITLE
mpicc: add page and Italian translation

### PR DESCRIPTION
**Pull request by mSamiolo**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.it/linux/mpicc.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# mpicc
+
+> Involucro Open MPI per il compilatore di C.
+> Shell che eseguono sul compilatore, aggiungono i relevanti argomenti e linkers necessari a compilare/collegare programmi Open MPI, invocando il sottostante compilatore di C per effettuare le effetive operazioni.
+
```
Comment (kbdharun) [inline pages.it/linux/mpicc.md:5]:
```suggestion
> Maggiori informazioni: <https://www.mpich.org/static/docs/latest/www1/mpicc.html>.

```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution and welcome to tldr. I have some suggestions for this page, feel free to apply them to the Italian page too.

**Comment by mSamiolo** [comment]
@kbdharun apologies, I am a noob on Git contributions! I am learning how it works

**Comment by kbdharun** [comment]
> @kbdharun apologies, I am a noob on Git contributions! I am learning how it works

No issues 😄

**Review by mSamiolo — COMMENTED** [review]
I agree with your modifications!

**Comment by mSamiolo** [comment]
What else it is needed to do to close the commit?

**Comment by mebeim** [comment]
@mSamiolo as I said above, those modifications need to be made to the page, along with the suggestions proposed by @kbdharun. We were just waiting for you on that. Do you want me to go ahead and apply the changes for you?

**Comment by mSamiolo** [comment]
I have update two commits with the modification you suggested. If you need further mods let me know

**Review by mebeim — APPROVED** [review]
I have applied the remaining changes myself, including removing an additional unrelated typo fix for `nmtui` (that can be done in another PR). Seems good to go now. Thanks again @mSamiolo :)

*Maintainer note: squash PR into a single commit when ready and clear the commit message bullet list as it is unneeded.*

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**